### ISSUE TITLE
An error occurred while inserting the rules into DOM. DOMException: Failed to execute 'insertRule' on 'CSSStyleSheet': Failed to parse the rule '.fela_sz:focus:-ms-input-placeholder{line-height:20px !important}'.

### ISSUE THREAD

**Issue by christophediprima** [rofrischmann/fela#634]
| Packages | Version |
| -------- | ------- |
| fela     | 6.2.3  |

-------------------
#### Description
Since I updated to the last versions of the fela packages, I have this kind of errors in my console:
```
insertRule.js:47 An error occurred while inserting the rules into DOM.  line-height:20px !important DOMException: Failed to execute 'insertRule' on 'CSSStyleSheet': Failed to parse the rule '.fela_sz:focus:-ms-input-placeholder{line-height:20px !important}'.
```

#### Steps to reproduce
Just tried to pass following structure to the renderer:
```javascript
{
':focus': {
    '::placeholder': {
         lineHeight:'20px !important'
      }
  }
}
```

**Comment by rofrischmann** [comment]
Which browser are you using?
Might be due to some new error messages that lead to false errors, we should remove those.

**Comment by christophediprima** [comment]
I am using Chrome 70

**Comment by christophediprima** [comment]
Also having it on latest Firefox

**Comment by rofrischmann** [comment]
Fixed with #635
